frame/evm/src/lib.rs

```diff
@@ -407,6 +409,7 @@ pub mod pallet {
 			max_priority_fee_per_gas: Option<U256>,
 			nonce: Option<U256>,
 			access_list: Vec<(H160, Vec<H256>)>,
+			authorization_list: Vec<(U256, H160, U256, Option<H160>)>,
```

Comment:
This should be: `(U64, H160, U256, bool, H256, H256)` to respect `[chain_id, address, nonce, y_parity, r, s]` **We need to validate the signatures**

```suggestion
			authorization_list: Option<Vec<(U256, H160, U256, MalleableTransactionSignature)>>,
```